test/e2e/framework/kubelet/stats.go

```diff
@@ -578,7 +578,7 @@ func (r *ResourceMonitor) GetMasterNodeLatest(usagePerNode ResourceUsagePerNode)
 		nodeAvgUsage[c].MemoryWorkingSetInBytes /= uint64(len(nodesUsage))
 		nodeAvgUsage[c].MemoryRSSInBytes /= uint64(len(nodesUsage))
 	}
-	result["master"] = masterUsage
+	result["controlplane"] = controlplaneUsage
```

Comment:
This likely doesn't exist unless you've updated result to have this as a possibility. Can you leave this as is or update result?